### ISSUE TITLE
Can vue-popper support vue3.0

### ISSUE THREAD

**Issue by qiuhuachuan** [RobinCK/vue-popper#141]
When can vue-popper support vue3.0

**Comment by asafyish** [comment]
Vue 3 final is out.

**Comment by johnenrick** [comment]
Any update?

**Comment by SherinBloemendaal** [comment]
I've rewritten the component completly in Vue 3.0 composition api style. I don't care about backwards compatibility to i won't create a PR. If you want the component, let me know.

**Comment by thomporter** [comment]
@SherinBloemendaal yes please!  I checked your repos, didn't see it in there, are you going to publish to NPM or great a git repo for it?

**Comment by SherinBloemendaal** [comment]
Yeah i think i am going to fork this library and publish it to NPM, i will let you guys know when its finished! I also upgraded the popperjs version to 2.0 (see: https://github.com/RobinCK/vue-popper/issues/127)

**Comment by whytspace** [comment]
@SherinBloemendaal I am very much interested in that rewrite of yours.

Did you already finish it? Can you publish on npm or share a github repo?

**Comment by SherinBloemendaal** [comment]
@whytspace Yes its finished. I still need to publish it but i don't have a lot of time. But, one of my interns need the vue 3.0 version so i will create it this week.

**Comment by gr8bit** [comment]
@SherinBloemendaal Interested big time!

**Comment by tadappnative** [comment]
> @whytspace Yes its finished. I still need to publish it but i don't have a lot of time. But, one of my interns need the vue 3.0 version so i will create it this week.

bla bla bla

**Comment by richard-viney** [comment]
Any update on Vue 3 support? Or an alternative library that would achieve a similar thing in Vue 3?

**Comment by valgeirb** [comment]
I've been working on a Vue 3 Popper ([vue3-popper](https://github.com/valgeirb/vue3-popper)) , it's early but please check it out and let me know what you think.

**Comment by richard-viney** [comment]
Thanks @valgeirb I will check it out. Would be good for people not to have to switch libraries when upgrading to Vue 3, though perhaps some of the APIs have changed enough that it's necessary? I've never actually used Vue 2.x so not entirely sure.

**Comment by SteiniDJ** [comment]
> Any update on Vue 3 support? Or an alternative library that would achieve a similar thing in Vue 3?

There's a similar component available for vue3 that might work for you, until this one is updated. https://github.com/valgeirb/vue3-popper

**Comment by craiggeil** [comment]
vue3-popper is missing too many properties, its really just a slimmed down version of vue-popper. Did the author ever create a vue 3 version or did they just disappear?

**Comment by RobinCK** [comment]
you can make a PR
